rust/src/edge.rs

```diff
@@ -555,42 +555,42 @@ impl SeleniumManager for EdgeManager {
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct EdgeProduct {
-    #[serde(rename = "Product")]
+    #[serde(rename = "Product", alias = "product")]
```

Comment:
Is it possible to "deserialize case insensitively" globally?

Reply:
I looked for a global solution, but unfortunately, the crate used for serialization (called serde) is not supported out of the box. It is not ideal, but the solution using an alias seems reasonable to me.